README.md

```diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 
 * Fry
 * Kestrel
+
+### .sh file instructions
+<details>
+     <summary>Clean.sh (Click to expand):</summary>
```

Comment:
```suggestion
    - clean.sh
```
we could use numbering or bullets here to keep the instructions for the two scripts distinct